firefox-ios/firefox-ios-tests/Tests/XCUITests/PhotonActionSheetTests.swift

```diff
@@ -49,25 +49,34 @@ class PhotonActionSheetTests: BaseTestCase {
         // User not logged in
         navigator.openURL(path(forTestPage: "test-mozilla-org.html"))
         waitUntilPageLoad()
+        // Temporarily disabled until url bar redesign work FXIOS-8172
+        /*
```

Comment:
This test from full functional test suite failed during overnight test run. Let me disable it for now.